Development environment setup/Linter setup.md

```diff
@@ -60,6 +60,11 @@ npm run lintJs
 npm run lintStyle
 ```
 
+On more recent projects we started using `bun` instead of `npm` as a package manager. The above commands will be the same, but you will use `bun` instead of `npm`, e.g.

```

Comment:
TBH we can just rewrite this as "we recommend bun, bun is good, use bun" 😄

Reply:
Just a note, `bun` does not require the `run` command